ui/js/component/publishForm/view.jsx

```diff
@@ -166,8 +167,8 @@ class PublishForm extends React.PureComponent {
 
   claim() {
     const { claimsByUri } = this.props;
-    const { uri } = this.state;
-    return claimsByUri[uri];
+    const { uri, publishedUri } = this.state;
+    return claimsByUri[uri] || claimsByUri[publishedUri];
```

Comment:
Can you help me understand what the underlying reason we need this additional value is? This seems like a potentially dangerous approach, and ideally edit would be initiated with just a single value (presumably the claim id).

Reply:
it's a fallback for `#600`, after re-publish you can't pass the `claim_id`